Paint storage

LaurenJeffersonPugh opened the pull request:
Created a very basic refill station and UI for the player paint tank.

Commit:
add paint storage

Commit:
tank bar colour fix

BertTDev commented on paint-prototype/Assets/Fork/Lauren/Player.cs:
```diff
@@ -0,0 +1,73 @@
+            }
+
+        }
+    }
```
Can we format one line ifs as:
if (boolean) function();
basically get rid of curly braces where not needed

BertTDev replied:
I was dumb - this functionality is in the player script

BertTDev commented on paint-prototype/Assets/Fork/Lauren/PaintTank.cs:
```diff
@@ -0,0 +1,29 @@
+        slider.value = health;
+        fill.color = gradient.Evaluate(slider.normalizedValue);
+
+    }
```
Add a removePaint method instead? instead of setting things it just removes from a set amount
Means we don't have to get the current amount of paint every time we want to add or remove.
Should also add an addPaint method for refilling

thomasscott95 reviewed (approved):
Work is only in Laurens scene to showcase tomorrow :)

Commit:
fixes

removed unneeded brackets + fixed health becoming less than 0/greater than the max

thomasscott95 reviewed (commented):
id would like to sit with you when your in to understand what some of the methods you are using to learn a little more :)
